### PR TITLE
Clean-up VacuumCommand

### PR DESCRIPTION
**Pull request by AFFogarty**
This PR makes two small changes to `VacuumCommand`.

* Remove `reservoirScheme` which is never used.
* ~~Make `isBloomFiltered` a `val` instead of `var` because it is never modified.~~

### CODE REVIEW

**Comment by databricks-cla-assistant** [comment]
[CLA assistant check](https://cla.dev.databricks.com/delta-io/delta?pullRequest=237) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.databricks.com/delta-io/delta?pullRequest=237) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dev.databricks.com/check/delta-io/delta?pullRequest=237) it.</sub>

**Comment by AFFogarty** [comment]
@imback82 @suhsteve @rapoth

File: src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala [hunk 1]
```diff
@@ -119,12 +119,11 @@ object VacuumCommand extends VacuumCommandImpl {
       val hadoopConf = spark.sparkContext.broadcast(
         new SerializableConfiguration(sessionHadoopConf))
       val basePath = fs.makeQualified(path).toString
-      var isBloomFiltered = false
```
Comment (tdas) [inline src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala]: Yeah this is probably an artifact of our code management. This is where we integrate with our internal bloom filter support. Can you please keep this as a var.

Comment (AFFogarty) [reply src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala]: Ok, changed back.

**Pull request closed without merging**